crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -50,26 +51,28 @@ class CrowdfundingProject(models.Model):
         [("validation", "Validation"), ("active", "Active")],
         required=True,
         default="validation",
+        readonly=True
     )
-    # event_type_id = fields.Many2one("event.type", "Event type")
+    owner_lastname = fields.Char(string="Your lastname")
+    owner_firstname = fields.Char(string="Your firstname")
 
-    # TODO fix event.type NULL error
-    # @api.model
-    # def create(self, vals):
-    #     res = super().create(vals)
-    #     res.event_id = self.env['crm.event.compassion'].create({
-    #         'name': vals.get('name'),
-    #         'event_type_id': self.env.ref(
-    #             "crowdfunding_compassion.event_type_crowdfunding").id,
-    #         'company_id': self.env.user.company_id.id,
-    #         'start_date': datetime.date.today(),
-    #         'end_date': vals.get('deadline'),
-    #         'hold_start_date': datetime.date.today(),
-    #         'number_allocate_children': vals.get('product_number_goal'),
-    #         'planned_sponsorships': vals.get('number_sponsorships_goal'),
-    #         'type': "crowdfunding"
-    #     })
-    #     return res
+    @api.model
```

Comment:
Did you succeed to fix your bug or did you uncomment by mistake ?

Reply:
I fixed it